code/HabitTracker/app/src/main/java/com/example/habittracker/utils/CustomDatePicker.java

```diff
@@ -61,6 +86,7 @@ public void onDateSet(android.widget.DatePicker view, int year, int month, int d
         myCalendar.set(Calendar.YEAR, year);
         myCalendar.set(Calendar.MONTH, month);
         myCalendar.set(Calendar.DAY_OF_MONTH, dayOfMonth);
+//        updateLabel();
```

Comment:
oh. I do need this though. I didn't think you had commented this out.... How else do you display the current date as a text to let the user know what they've chosen is selected?

Reply:
Line 90 should display current date for you:
updateLabel(myCalendar.getTime());